FreeRTOS/FreeRTOS
# Maintenance -- clean up readme.txt and add url to GitHub.

Description
-----------
Changes:
- remove readme.txt
- add missing FAQ information into README.md
- add GitHub-FreeRTOS-Home.url

Test Steps
-----------
Visually looks fine. URL is working fine with Safari as default browser.

Related Issue
-----------
N/A


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
